extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkSQLLineageParseHelper.scala

```diff
@@ -327,6 +336,19 @@ trait LineageParser {
           joinColumnsLineage(parentColumnsLineage, getSelectColumnLineage(p.aggregateExpressions))
         p.children.map(extractColumnsLineage(_, nextColumnsLineage)).reduce(mergeColumnsLineage)
 
+      case p: Expand =>
+        val childColumnsLineage = ListMap(p.output.collect {
+          case attr
+              if p.references.find(
+                _.name == attr.name.split('.').last.stripPrefix("`").stripSuffix("`")).nonEmpty =>
+            val ref = p.references
+              .find(_.name == attr.name.split('.').last.stripPrefix("`").stripSuffix("`")).get
+            (attr, ref.references)
```

Comment:
what happens with `select count(distinct c1 + c2) as x, count(distinct(c3)) as y` ?
do we return x -> [c1, c2] and y -> [c3] ?

Reply:
yes, i fix it